### ISSUE TITLE
why i got torch.Size([701, 512]) torch.Size([51355, 512]) 514 Recall@1:0.00 Recall@5:0.14 Recall@10:0.14 Recall@top1:0.86 AP:0.35

### ISSUE THREAD

**Issue by samcheung1** [layumi/University1652-Baseline#39]
I used three_view_long_share_d0.75_256_s1_google model and used test.py, but i got torch.Size([701, 512]) torch.Size([51355, 512]) 514 Recall@1:0.00 Recall@5:0.14 Recall@10:0.14 Recall@top1:0.86 AP:0.35.
That was so bad, so i want to get why?

**Comment by samcheung1** [comment]
The  training loss & accuracy in train.py is normal, but i am not sure the training loss & accuracy in test.py is normal. Can you teach me how?

**Comment by layumi** [comment]
Hi @samcheung1 

1. You may check the loss value and accuracy value during training. The accuracy rate should converge to 90+% after training.  
2. Could you print the query and gallery feature in this line?
https://github.com/layumi/University1652-Baseline/blob/master/test.py#L245-L246

**Comment by samcheung1** [comment]
Thank you. I got it.

---Original---
From: "Zhedong ***@***.***&gt;
Date: Mon, Mar 13, 2023 22:40 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [layumi/University1652-Baseline] why i got torch.Size([701, 512])torch.Size([51355, 512]) 514 ***@***.***:0.00 ***@***.***:0.14 ***@***.******@***.***:0.86 AP:0.35 (Issue #39)




 
Hi @samcheung1
  
You may check the loss value and accuracy value during training. The accuracy rate should converge to 90+% after training.
 
Could you print the query and gallery feature in this line?
 https://github.com/layumi/University1652-Baseline/blob/master/test.py#L245-L246
